cfgmgr/coppmgr.cpp

```diff
@@ -352,15 +395,39 @@ CoppMgr::CoppMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, c
 
         if (!trap_group_fvs.empty())
         {
+            if (isDupEntry(i.first, trap_group_fvs))
+            {
+                continue;
+            }
             m_appCoppTable.set(i.first, trap_group_fvs);
         }
+
         setCoppGroupStateOk(i.first);
         auto g_cfg = std::find(group_cfg_keys.begin(), group_cfg_keys.end(), i.first);
         if (g_cfg != group_cfg_keys.end())
         {
             g_copp_init_set.insert(i.first);
         }
     }
+
+    // Delete unsupported keys from preserved copp tables
+    unordered_set<string> supported_copp_keys;
+    for (auto i : m_coppGroupInitCfg)
+    {
+        supported_copp_keys.emplace(i.first);
```

Comment:
Can't the supported_copp_keys be filled inside the if loop in 397 when setting the app_db table?